packages/adapter-node/index.js

```diff
@@ -10,22 +10,30 @@ const pipe = promisify(pipeline);
 const files = fileURLToPath(new URL('./files', import.meta.url).href);
 
 /** @type {import('.')} */
-export default function ({
-	out = 'build',
-	precompress,
-	env: {
-		path: path_env = 'SOCKET_PATH',
-		host: host_env = 'HOST',
-		port: port_env = 'PORT',
-		origin: origin_env = 'ORIGIN',
-		xffDepth: xff_depth_env = 'XFF_DEPTH',
-		headers: {
-			address: address_header_env = 'ADDRESS_HEADER',
-			protocol: protocol_header_env = 'PROTOCOL_HEADER',
-			host: host_header_env = 'HOST_HEADER'
+export default function (opts = {}) {
+	// TODO remove for 1.0
+	// @ts-expect-error
+	if (opts.env) {
+		throw new Error(
+			`options.env has been removed in favour of options.environment, which has different properties. Consult the adapter-node README: https://github.com/sveltejs/kit/tree/master/packages/adapter-node`
+		);
+	}
+
+	const {
+		out = 'build',
+		precompress,
+		environment: {
```

Comment:
would something like `envAliases` make sense as a name?

Reply:
i feel like to be properly accurate it would need to be `envVarAliases` which seems like a step too far